app/routes/admin/content/translations.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
     return [
       {
         code      : 'en',
-        sourceUrl : '#'
+        sourceUrl : this.get('loader').load('/admin/content/translations/all')
```

Comment:
this will actually execute the loading of the file. You don't need to pass this in the model.. just execute this when the download button is pressed.